templates/store-lists-loop.php

```diff
@@ -13,13 +13,14 @@
                     $store_phone              = $vendor->get_phone();
                     $store_info               = dokan_get_store_info( $seller->ID );
                     $store_address            = dokan_get_seller_short_address( $seller->ID );
-                    $store_banner_url         = $store_banner_id ? wp_get_attachment_image_src( $store_banner_id, $image_size ) : DOKAN_PLUGIN_ASSEST . '/images/default-store-banner.png';
+                    $default_store_banner     = DOKAN_PLUGIN_ASSEST . '/images/default-store-banner.png';
```

Comment:
there is no need for the $default_store_banner, since we've provided a default value under settings, this line is not required.